types/mathjs/mathjs-tests.ts

```diff
@@ -1,415 +1,437 @@
-import { create, factory, all, MathJsFunctionName } from 'mathjs';
```

Comment:
Are all of the changes in this file whitespace changes?

Reply:
Most are formatting except for the last snippet (from new file L507), which is a new test case for the dependency typings.